lib/apis/moray.js

```diff
@@ -145,12 +138,15 @@ Moray.prototype.connect = function () {
             log.error(err, 'moray client error');
         });
 
+        self.log.info('Setting up buckets...');
+
         self._setupBuckets(function (err) {
             if (err) {
-                self.log.error({ err: err }, 'Buckets were not loaded');
+                self.emit('error', err);
+                self.log.error('Error when setting up moray buckets: ' + err);
```

Comment:
You should pass the error object as the first argument, e.g. `self.log.error(err, 'failed to set up moray buckets')`.  I would also do this _before_ emitting the error through the `EventEmitter`.